### PR TITLE
fix materialize override

### PR DESCRIPTION
**Pull request by jdanyow**

### CODE REVIEW

**Comment by jdanyow** [comment]
the original override has `"files": ["dist"],` which results in empty directories

File: package-overrides/github/Dogfalo/materialize@0.97.0.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "files": [
-    "dist"
-  ],
-  "main": "dist/js/materialize",
+  "main": "js/materialize",
+  "directories": {
+    "lib": "dist"
```
Comment (guybedford) [inline package-overrides/github/Dogfalo/materialize@0.97.0.json]: We avoid adding this property to overrides

**Comment by guybedford** [comment]
Thanks for posting. Do you mean the directories apart from `dist` are empty, or is there an issue with directories being removed unnecessarily?

**Comment by jdanyow** [comment]
the subdirectories were empty.  The install would create the directory structure but there would be no files within.  Maybe the `files` option doesn't do partial matches?  In any case, I think using the directories/lib option yields a better install result.

**Comment by guybedford** [comment]
`files: ['dist']` will clear out all directories other than `dist`. If it was clearing anything inside `dist` that would be a bug though. `directories.lib` is a breaking change so we can't do that unfortunately.

**Comment by jdanyow** [comment]
hmm- this came up because a user on stackoverflow hit the empty directory issue, which I reproduced when testing.  See comment [here](http://stackoverflow.com/questions/33614451/materialize-css-javascript-functions-not-found-in-aurelia-project/33632241?noredirect=1#comment55038054_33632241)

If you try the current override do you hit the same issue?  Maybe I'm doing something wrong.

**Comment by guybedford** [comment]
The issue here is that it is using the release archive at https://github.com/Dogfalo/materialize/releases, which does not contain a `dist` folder.

So I think the correction here will work fine as you've posted without the `directories.lib` part.

**Pull request closed without merging**